en/ros/mavros_installation.md

```diff
@@ -32,7 +32,7 @@ Kinetic also supports Debian Jessie amd64 and arm64 (ARMv8).
 Use `apt-get` for installation:
 
 ```
-sudo apt-get install ros-kinetic-mavros ros-kinetic-mavros-extras
+sudo apt-get install ros-${ROS_DISTRO}-mavros ros-${ROS_DISTRO}-mavros-extras ros-${ROS_DISTRO}-mavros-msgs
```

Comment:
Could you also update the instruction so that this also works for ROS `Noetic`? Here: https://github.com/PX4/PX4-user_guide/pull/2383/files#diff-481de206f6c6e77b8c8e5ddba6c64e280a1dbbce871f1c9e699e420e744a6503R21